guides/common/modules/con_installing-and-configuring-advisorengine.adoc

```diff
@@ -0,0 +1,13 @@
+= Installing and configuring {advisorengine}
```

Comment:
```suggestion
[id="installing-and-configuring-advisorengine"]
= Installing and configuring {advisorengine}
```

Either this, or we'll need a new attribute because the value for Satellite contains spaces: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L46